AppCreationScripts/sample.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Sample": {
+      "Title": "An Angular single-page application that authenticates users with Azure AD and calls a protected ASP.NET Core web API",
+      "Level": 200,
+      "Client": "TodoListSPA",
+      "Service": "TodoListAPI",
+      "RepositoryUrl": "ms-identity-javascript-angular-spa-aspnetcore-webapi",
+      "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+      {
+        "Id": "service",
+        "Name": "TodoListAPI",
+        "Kind": "WebApi",
+        "Audience": "AzureADandPersonalMicrosoftAccount",
+        "HomePage": "https://localhost:44351/api/todolist/",
+        "ManualSteps": [
+          {
+            "Comment": "Navigate to the Manifest page and change 'signInAudience' to 'AzureADandPersonalMicrosoftAccount'."
+          }
+        ]
+      },
+      {
+        "Id": "client",
+        "Name": "TodoListSPA",
+        "Kind": "SinglePageApplication",
+        "Audience": "AzureADandPersonalMicrosoftAccount",
+        "HomePage": "http://localhost:4200/",
```

Comment:
>http://localhost:4200/ [](start = 21, length = 22)

Just checking, is there no way to use https here?

Reply:
the certificates has to be manually set and configured (unlike .net) -I think it might be too much for the sample and distract from the main topic : (